Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/platform/PlatformGamesView.kt

```diff
@@ -0,0 +1,31 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+package org.dolphinemu.dolphinemu.ui.platform
+
+/**
+ * Abstraction for a screen representing a single platform's games.
+ */
+interface PlatformGamesView {
+    /**
+     * Pass a click event to the view's Presenter. Typically called from the
+     * view's list adapter.
+     *
+     * @param gameId The ID of the game that was clicked.
+     */
+    fun onItemClick(gameId: String) { /* Empty default impl */ }
```

Comment:
Is there a point to having an empty default implementation?

Reply:
I don't have to override it in `PlatformGamesFragment` if it has a default implementation. Since it was doing nothing in the first place, this can just make things a little cleaner. But if you'd like I could just delete it.

Reply:
We can keep it like this.